### ISSUE TITLE
Add SDG color variables: 2-5, 8-11, 13, 16, 17

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#4233]
### Overview
We need to add the SDG (Sustainable Development Goal) colors to our repo so we can use these colors with their corresponding, matching SDG icons.

### Action Items
- [x] In the `_sass/variables`, create a new file: `_colors-sdg.scss`
- [x] In the file you just created, add the following:
```
/**
 * Sustainable Development Goal (SDG) Colors
 *
 * The hex color values for these variables match the SDG color icons provided by the United Nations (UN).
 * These hex values are from the last the page in the SDG Guidelines provided by the UN:
 * https://www.un.org/sustainabledevelopment/wp-content/uploads/2019/01/SDG_Guidelines_AUG_2019_Final.pdf
 */

$color-sdg2: #DDA63A;
$color-sdg3: #4C9F38;
$color-sdg4: #C5192D;
$color-sdg5: #FF3A21;
$color-sdg8: #A21942;
$color-sdg9: #FD6925;
$color-sdg10: #DD1367;
$color-sdg11: #FD9D24;
$color-sdg13: #3F7E44;
$color-sdg16: #00689D;
$color-sdg17: #19486A;
``` 
- [x] In the `_sass/main.scss`, under the `General Settings and Utilities` section, add this:
```
@import 'variables/colors-sdg';
```

#### Merge Team, Technical Lead, or Product Manager
- [ ] Once the pull request associated with this issue is merged, go to each of the following issues and check if all the issues/dependencies listed under the `Dependency` section are closed or checked off. If the dependencies are all closed or checked, move those issues into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it.
  - [x] #4344 

### Resources/Instructions
- [_sass/variables](https://github.com/hackforla/website/tree/gh-pages/_sass/variables)
- [_sass/main.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/main.scss)
- [SDGs_HEX_Colors tab in HfLA: Projects: SDG and Targets Google Sheet](https://docs.google.com/spreadsheets/d/1lV8d8AMSJ5HOiTWlFE7lzbqiKxqMmh8pHADcJMdp7JE/edit#gid=741671421)
- This issue is tracked in #4153

**Comment by JessicaLucindaCheng** [comment]
Before I finalize this issue, I'm wondering
- Should the SDG colors be considered part of the Design System? Ans: From Bonnie, yes but like a special case
  - If no, then   
    - Should we have a separate file for SDG color variables, such as `_sdg-colors.scss`? Ans: We can
    - Should we have a separate file for SDG background color classes, such as `_sdg-background-colors.scss`?
- How do we want to name the colors? Did we pick a convention/website to follow?
  - I was just going to use the color names in the last page of the [UN's SDG guide](https://www.un.org/sustainabledevelopment/wp-content/uploads/2019/01/SDG_Guidelines_AUG_2019_Final.pdf) and do the following:
  - UN provided color for SDG 11: `golden yellow`
    - Color variable: `$color-sdg-goldenyellow` 
    - Background class: `.background-sdg-goldenyellow { background: $color-sdg-goldenyellow; }`

I will bring it up in Sun, Mar 26 All Team Meeting or Mon, Mar 27 Dev/PM Meeting.


Edit: 
- From Bonnie, we can have a new variable SCSS file just for SDG color variables.
- Explain why we can't use the color-COLORNAME classes for background colors.
  -  Example of color-COLORNAME classes: https://github.com/hackforla/website/blob/f6df44e3200b477a753475264c945ef7d3e1b268/_sass/elements/_color-styles.scss#L53-L55
      - The SCSS `color` property specifies the color of the text (definition from https://www.w3schools.com/cssref/pr_text_color.php)
  - Background colors in SCSS: https://github.com/hackforla/website/blob/f6df44e3200b477a753475264c945ef7d3e1b268/_sass/elements/_cards.scss#L51-L68 
  - Code examples of its application:
    - https://github.com/hackforla/website/blob/f6df44e3200b477a753475264c945ef7d3e1b268/_includes/forms/contact-us.html#L5
    - https://github.com/hackforla/website/blob/f6df44e3200b477a753475264c945ef7d3e1b268/_includes/forms/contact-us.html#L19

**Comment by StephenTheDev1001** [comment]
Availability: Sun-Thrus 3pm-7pm PDT.
ETA: Apr 6th
